interface-definitions/include/isis/protocol-common-config.xml.i

```diff
@@ -492,6 +492,61 @@
         </node>
       </children>
     </node>
+    <node name="ipv6">
+      <properties>
+        <help>Redistribute IPv6 routes</help>
+      </properties>
+      <children>
+        <node name="bgp">
+          <properties>
+            <help>Border Gateway Protocol (BGP)</help>
```

Comment:
Shouldn't this read:

Redistribute BGP routes into IS-IS?

Reply:
Ahh, yes that is a better way to describe it. Will make that change. You're right.

Thank you.